File: Selenium/Activities/Activity2.java
package com.activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
	public static void main(String[] args) throws Exception {

		WebDriver driver = new FirefoxDriver();

		// Open the web page
		driver.get("https://training-support.net/webelements/login-form");

		// Print the title of the page
		System.out.println("Title of page is: " + driver.findElement(By.tagName("h1")).getText());

		// Find the username field and enter the username
		System.out.println("Entering username..");
		driver.findElement(By.id("username")).sendKeys("admin");
		// Find the password field and enter the password
		System.out.println("Entering password..");
		driver.findElement(By.id("password")).sendKeys("password");
		// Find the login button and click it
		System.out.println("Clicking Submit..");
		driver.findElement(By.xpath("//button[text()='Submit']")).click();

		Thread.sleep(3000);
		// Print the confirmation message
		System.out.println(driver.findElement(By.tagName("h1")).getText());
		System.out.println(driver.findElement(By.tagName("h2")).getText());

		// Close the browser
		driver.quit();
	}
}

File: Selenium/Activities/activity2.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
import time
# Start the Driver
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("https://training-support.net/webelements/login-form")

    time.sleep(2)
    # Print the title of the page
    print("Page title is: ", driver.title)

    # Find the username field
    username = driver.find_element(By.ID, "username")
    # Find the password field
    password = driver.find_element(By.ID, "password")

    # Enter the given credentials
    # Enter username
    username.send_keys("admin")
    # Enter password
    password.send_keys("password")

    # Find the login button
    login = driver.find_element(By.XPATH, "//button[text()='Submit']")
    login.click()
    time.sleep(2)
    # Print the login message
    message1 = driver.find_element(By.TAG_NAME, "h1")
    print("Login message: ", message1.text)
    message2 = driver.find_element(By.TAG_NAME, "h2")
    print("Login message: ", message2.text)